cpu/arm7_common/include/pm_conf.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2017 TU Braunschweig, Robert Hartung
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @brief           CPU specific power management options
+ *
+ * @author          Robert Hartung <hartung@ibr.cs.tu-bs.de>
+ */
+
+#ifndef PM_CONF_COMMON_H
+#define PM_CONF_COMMON_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Power management configuration
+ *
+ * Select the required implementations from the common CPUs or fallbacks
+ * @{
+ */
+#define NEEDS_ARM7_PM_REBOOT
```

Comment:
might be nitpicky, but I would definitely prefer `NEEDS_PM_ARM7_REBOOT` to keep `pm` as prefix and make clear that this is a `pm` specific option